apps/fz_http/lib/fz_http_web/live/user_live/vpn_status_component.ex

```diff
@@ -0,0 +1,24 @@
+defmodule FzHttpWeb.UserLive.VPNStatusComponent do
+  @moduledoc """
+  Handles VPN status tag.
+  """
+  use FzHttpWeb, :live_component
+
+  @impl Phoenix.LiveComponent
+  def render(assigns) do
+    ~H"""
+    <label>
```

Comment:
Hmm, probably shouldn't be a label? A label is almost always used to wrap or alongside an input. It's clickable.